cli/flags.rs

```diff
@@ -995,17 +998,27 @@ fn eval_subcommand<'a, 'b>() -> App<'a, 'b> {
   deno eval \"console.log('hello world')\"
 
 To evaluate as TypeScript:
-  deno eval -T \"const v: string = 'hello'; console.log(v)\"
+  deno eval --ext=ts \"const v: string = 'hello'; console.log(v)\"
 
 This command has implicit access to all permissions (--allow-all).",
     )
     .arg(
+      // TODO(@satyarohith): remove this argument in 2.0.
       Arg::with_name("ts")
         .long("ts")
         .short("T")
         .help("Treat eval input as TypeScript")
         .takes_value(false)
-        .multiple(false),
+        .multiple(false)
+        .hidden(true),
+    )
+    .arg(
+      Arg::with_name("ext")
+        .long("ext")
+        .help("Set standard input (stdin) content type")
+        .takes_value(true)
+        .default_value("js")
+        .possible_values(&["ts", "js"]),
```

Comment:
Shouldn't `jsx` and `tsx` be allowed as well? 🤔

Reply:
The current implementation doesn't have support for `jsx`/`tsx` so I didn't include them in the possible values. 

But it's very easy (just return the appropriate media type and I already have the implemented version locally) to add the support. I'm +1 for supporting them. Let me your thoughts.

Reply:
I added support for them in the new changes